server/index.ts

```diff
@@ -1,8 +1,38 @@
 import Express from 'express';
+const { router } = require('./router');
+const cors = require('cors');
+import {auth} from 'express-openid-connect'
+require("dotenv").config();
+
+
+const config = {
+  authRequired: false,
+  auth0Logout:true,
+  secret: process.env.AUTH0_SECRET,
```

Comment:
Can we have fallback values here?

Reply:
The auth0 client and auth0 domain are specific values from the auth0 application account

Reply:
Added fallbacks on secret and url